### ISSUE TITLE
When a new issue assigned or any comments coming, the assignee should also be emailed.

### ISSUE THREAD

**Issue by yourchanges** [gogs/gogs#4220]
When a new issue assigned or any comments coming, the assignee should also be emailed.

before https://github.com/gogits/gogs/blob/master/models/issue_mail.go#L123 , add following code:
```
//keep the Assignee as the same role as the watchers
	if issue.Assignee != nil {
		if com.IsSliceContainsStr(names, issue.Assignee.Name) == false {
			tos = append(tos, issue.Assignee.Email)
			names = append(names, issue.Assignee.Name)
		}
	}
```

any mistake or omission?

**Comment by Unknwon** [comment]
This is claimed to be fixed by merging #5628, please help test on `develop` branch or https://try.gogs.io.

**Comment by hochhaus** [comment]
Tested and confirmed working via https://try.gogs.io/. Thanks for the quick merge.
